.github/workflows/rebase-api.yml

```
@@ -0,0 +1,68 @@
+            core.setOutput('baseRef', pr.base.ref); // The base branch name, e.g., main
+            core.setOutput('headSha', pr.head.sha); // The sha to checkout
+            console.log(`PR #${prNumber}: Head ref is ${pr.head.ref} (${pr.head.sha}), Base ref is ${pr.base.ref}`);
+            return pr.head.ref; # Return value not strictly needed but good practice
+
+      # Checkout the correct SHA after potentially checking out the wrong thing initially
+      - name: Checkout Correct PR Head SHA
+        run: git checkout ${{ steps.get_pr_data.outputs.headSha }}
+
+      - name: Run Rebase API Action
+        id: rebase_api
+        uses: ./.github/actions/rebase-api
```

actionlint (1.7.4)

file "dist/index.js" does not exist in "/home/jailuser/git/.github/actions/rebase-api". it is specified at "main" key in "runs" section in "Rebase API Changes" action

(action)